doc/man/scons.xml

```diff
@@ -4148,53 +4153,50 @@ vars.Save('variables.cache', env)
   </varlistentry>
 
   <varlistentry>
-  <term>vars.GenerateHelpText(<emphasis>env</emphasis>, [<emphasis>sort</emphasis>])</term>
+  <term><replaceable>vars</replaceable>.<function>GenerateHelpText</function>(<parameter>env, [sort]</parameter>)</term>
   <listitem>
-<para>This generates help text documenting the customizable construction
-variables suitable to passing in to the Help() function.
-<emphasis>env</emphasis>
+<para>Generate help text documenting the customizable construction
+variables, suitable for passing in to the &f-link-Help; function.
+<parameter>env</parameter>
 is the &consenv; that will be used to get the actual values
-of customizable variables. Calling with
-an optional
-<emphasis>sort</emphasis>
-function
-will cause the output to be sorted
-by the specified argument.
-The specific
-<emphasis>sort</emphasis>
-function
-should take two arguments
-and return
--1, 0 or 1
-(like the standard Python
-<emphasis>cmp</emphasis>
-function).
+of the customizable variables. If the (optional)
+value of <parameter>sort</parameter>
+is callable, it is used as a comparison function to
+determine how to sort the added variables.
+This function must accept two arguments, compare them,
+and return a negative integer if the first is
+less-than the second, zero for equality, or a positive integer
+for greater-than.
 
-Optionally a Boolean value of True for <emphasis>sort</emphasis> will cause a standard alphabetical sort to be performed</para>
+Optionally a Boolean value of <constant>True</constant>
+for <parameter>sort</parameter> will cause a standard
+alphabetical sort to be performed.</para>
 
 <programlisting language="python">
 Help(vars.GenerateHelpText(env))
+
+def cmp(a, b):
+    return (a &gt; b) - (a &lt; b)
+
 Help(vars.GenerateHelpText(env, sort=cmp))
 </programlisting>
 
   </listitem>
   </varlistentry>
 
   <varlistentry>
-  <term>FormatVariableHelpText(<emphasis>env</emphasis>, <emphasis>opt</emphasis>, <emphasis>help</emphasis>, <emphasis>default</emphasis>, <emphasis>actual</emphasis>)</term>
+  <term><replaceable>vars</replaceable>.<function>FormatVariableHelpText</function>(<parameter>env, opt, help, default, actual</parameter>)</term>
   <listitem>
-<para>This method returns a formatted string
+<para>Return a formatted string
```

Comment:
Returns?

Reply:
Sure.  The docs are not consistent, but I can use "returns" when I make changes... to be more helpful, doc systems like Sphinx consider return and returns as synonyms, nice if somebody took a stand :)

Reply:
note the other function descriptions in this section use "Return a", and most of the rest of the manpage function descriptions use "Returns a"... did you want that change applied a little more widely (limiting the change to the Variables section)?

Reply:
Changed.